modules/ROOT/pages/collaborating-with-your-api-portal-community.adoc

```diff
@@ -0,0 +1,30 @@
+= Collaborating With Your API Portal Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+API Experience Hub uses Salesforce Chatter Feed components to enable collaboration with members of an API portal community by:
```

Comment:
Put user value first?
You can enable .... using the Salesforce Chatter in API....